Tests/test_PDB_MMCIF2Dict.py

```diff
@@ -26,37 +26,305 @@
+                                                       "20.974", "20.796", "28.370",
+                                                       "29.565", "21.248", "25.744",
+                                                       "8.691", "30.789", "30.905",
+                                                       "28.623", "24.935", "23.462",
+                                                       "9.924", "28.729", "13.579",
+                                                       "23.652", "25.631", "17.799",
+                                                       "23.547", "16.363", "24.125",
+                                                       "33.063", "29.209", "10.391",
+                                                       "12.221", "18.997", "16.360",
+                                                       "27.915", "28.158", "21.975",
+                                                       "27.069", "30.148", "21.196",
+                                                       "8.864", "13.228", "18.577",
+                                                       "20.526", "25.758", "7.838",
+                                                       "20.569", "13.009", "19.229",
+                                                       "17.655", "30.445", "9.014",
+                                                       "3.398", "31.603", "16.543",
+                                                       "12.037", "7.261", "5.607",
+                                                       "23.532", "30.701", "32.300",
+                                                       "34.351", "9.450", "29.476",
+                                                       "13.681", "26.728", "10.004",
+                                                       "30.553", "23.569", "10.927",
+                                                       "17.983", "8.191", "32.095",
+                                                       "11.520", "13.249", "15.919",
+                                                       "11.187", "16.743"])
+        # Turn black code style on
+        # fmt: on
         self.assertEqual(
             mmcif["_struct_ref.pdbx_seq_one_letter_code"],
-            [textwrap.dedent("""\
+            [
+                textwrap.dedent(
+                    """\
             GARASVLSGGELDKWEKIRLRPGGKKQYKLKHIVWASRELERFAVNPGLLETSEGCRQILGQLQPSLQTGSEELRSLYNT
             IAVLYCVHQRIDVKDTKEALDKIEEEQNKSKKKAQQAAADTGNNSQVSQNYPIVQNLQGQMVHQAISPRTLNAWVKVVEE
             KAFSPEVIPMFSALSEGATPQDLNTMLNTVGGHQAAMQMLKETINEEAAEWDRLHPVHAGPIAPGQMREPRGSDIAGTTS
             TLQEQIGWMTHNPPIPVGEIYKRWIILGLNKIVRMYSPTSILDIRQGPKEPFRDYVDRFYKTLRAEQASQEVKNWMTETL
             LVQNANPDCKTILKALGPGATLEEMMTACQGVGGPGHKARVLAEAMSQVTNPATIMIQKGNFRNQRKTVKCFNCGKEGHI
             AKNCRAPRKKGCWKCGKEGHQMKDCTERQANFLGKIWPSHKGRPGNFLQSRPEPTAPPEESFRFGEETTTPSQKQEPIDK
-            ELYPLASLRSLFGSDPSSQ""")])
+            ELYPLASLRSLFGSDPSSQ"""
+                )
+            ],
+        )
```

Comment:
Perhaps implicit string concatenation would be better than tripled quoted with dedent?

Reply:
Hmm - I hadn't appreciated that this way the lines of content would need two lines each - if we ignore that rule we'd have 99 character lines.

What do you think, was the dedent version better - even with 92 character lines? Or any other ideas?

(This makes me wonder why does ``pdbx_seq_one_letter_code`` have the embedded line breaks in the first place?)